static/scripts/oppia-player-0.0.0.js

```diff
@@ -208,6 +210,9 @@
     }
 
     var iframe = document.createElement('iframe');
+    iframe.isEmbedded = true;
```

Comment:
Now it makes much more sense that all this files are under a "static" folder... sorry for that. I'm reverting the changes!